### PR TITLE
Trashed reply restore fix

### LINKED ISSUE #2441: Discussion: Trashed reply can't be restore
**Describe the bug**
When a reply put in a trash and trying to restore, the reply will eventually disappear in discussion.

**To Reproduce**
1. Go to any discussion.
2. Trash any reply.
3. Restore it.
4. Instead of showing, the reply will disappear.

**Expected behavior**
Trash reply in discussion should appear when trying to restore.

**Screenshots**
Demo Site: https://www.loom.com/share/ab57f1b5912e454fb74e749925d2772d
Local Site: https://www.loom.com/share/200975e6e0de49be88f9636f0a1de9d5

**Support ticket links**
https://secure.helpscout.net/conversation/1525917832/144514

### PR DESCRIPTION
**Pull request by ngolwalkar**
### Changes proposed in this Pull Request:

Fixes #2441

### How to test the changes in this Pull Request:

1. Go to any discussion.
2. Trash any reply.
3. Restore it.
4. Now reply will be restored.

### Proof Screenshots or Video

https://www.loom.com/share/bfeaca6bbcac4141a15b3a97c012684b

### CODE REVIEW

File: src/bp-forums/replies/functions.php [hunk 1]
```diff
@@ -1910,13 +1910,19 @@ function bbp_trashed_reply( $reply_id = 0 ) {
...
 	$reply_id = bbp_get_reply_id( $reply_id );
 
 	if ( empty( $reply_id ) || ! bbp_is_reply( $reply_id ) ) {
 		return false;
 	}
 
+	$update_reply = array(
+		'ID' 			=> $reply_id,
+		'post_status'	=>	$previous_status
+	);
```
Comment (chetansatasiya) [inline src/bp-forums/replies/functions.php:1923]: @ngolwalkar make the code proper alligned

**Review by chetansatasiya — CHANGES_REQUESTED** [review]
Need code formatted as per PHPCS

File: src/bp-forums/replies/functions.php (hunk 1 above)
Comment (ngolwalkar) [reply src/bp-forums/replies/functions.php]: @chetansatasiya It is done now. Please check.

**Review by chetansatasiya — APPROVED** [review]

**Comment by mohit-buddyboss** [comment]
@ngolwalkar 

FIXED>Tested on dev branch of the platform.

Tested for Group discussion replies and for individual forum discussion replies. Also, verified Trashed and Restore functionality from front end as well as from back end. This functionality is working fine and replies are getting trashed and restore.

Video: https://www.loom.com/share/a59516d79568435a9eac5a20547d16da

Related issue is resolve now and PR is ready for release.

cc @chetansatasiya @sourabhmatolia

**Pull request merged**